aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  MaskedInput,
+  Header,
+  Heading,
+  Text,
+  TextInput,
+} from 'grommet';
+import { Apple, CreditCard } from 'grommet-icons';
+import { FormContainer } from '.';
+import { creditCardMask, cvvValidation, dateMask } from './formHelpers';
+
+export const PayExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Pay
+          </Heading>
+          <Text>for your HPE products</Text>
+        </Header>
+        <Box
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box>
+              <Button
+                label="Pay"
+                type="button"
+                primary
+                color="text-weak"
+                icon={<Apple />}
+                gap="xxsmall"
+                size="large"
+              />
+            </Box>
+            <Box margin="small" align="center">
+              <Text color="text-xweak">or</Text>
+            </Box>
+            <Heading level={4} margin={{ bottom: 'small', top: 'none' }}>
+              Credit Card Information
+            </Heading>
+            <FormField name="cardName" htmlFor="cardName" label="Name on Card">
+              <TextInput
+                id="cardName"
+                name="cardName"
+                placeholder="Jane Smith"
+              />
+            </FormField>
+            <FormField name="cardNumber" label="Credit Card Number">
```

Comment:
```suggestion
            <FormField htmlFor="cardNumber" name="cardNumber" label="Credit Card Number">
```

We need the `htmlFor` prop in order to properly tie the label to the input. It should be the same value in the input's id